src/wazuh_db/wdb_sca.c

```diff
@@ -135,7 +135,8 @@ int wdb_sca_find(wdb_t * wdb, int pm_id, char * output) {
 }
 
 /* Insert configuration assessment entry. Returns 0 on success or -1 on error (new) */
-int wdb_sca_save(wdb_t * wdb, int id,int scan_id,char * title,char *description,char *rationale,char *remediation, char * file,char * directory,char * process,char * registry,char * reference,char * result,char * policy_id,char * command,char *status,char *reason) {
+int wdb_sca_save(wdb_t * wdb, int id, int scan_id, char * title, char *description, char *rationale, char *remediation, char *condition, char * file,
```

Comment:
Seize the opportunity to reformat all parameters to `type *variable`, removing the space between the `*` and the variable.
Do the same if find the same inconsistence in any line changed by this PR.